xs/src/libslic3r/ConfigBase.cpp

```diff
@@ -16,7 +17,11 @@
 #include <boost/config.hpp>
 #include <boost/foreach.hpp>
 #include <boost/lexical_cast.hpp>
+#ifdef BOOST_NOWIDE_FOUND
+#include <boost/nowide/cstdlib.hpp>
+#else
 #include <boost/nowide/cenv.hpp>
+#endif
```

Comment:
I might have screwed up the rebase, but here it fails with:

```
src/libslic3r/Config.cpp:20:10: fatal error: boost/nowide/cenv.hpp: No such file or directory
   20 | #include <boost/nowide/cenv.hpp>
      |          ^~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
```

Reply:
The attached 4996.patch above modifies that file, but the spec file uses https://github.com/slic3r/Slic3r/pull/4996.patch and that one doesn't modify `src/libslic3r/Config.hpp`

Reply:
The url in spec is fake, I was not doing it properly.